config.js

```diff
@@ -76,7 +77,7 @@ var loadSettings = function(callback) {
 
 var loadTranslations = function() {
   var options = { key: [ 'translations', true ], include_docs: true };
-  db.medic.view('medic', 'doc_by_type', options, function(err, result) {
+  db.medic.view('medic-client', 'doc_by_type', options, function(err, result) {
```

Comment:
Again this looks like a non-client using the client-specific ddoc.  But again, I may be misunderstanding the use of the word "client" in this context.